webview-ui/connectionsetup.js

```diff
@@ -279,6 +290,36 @@ function populateTokenField(token) {
   saveState();
 }
 
+function replaceOrganizationDropdown(organizations) {
+  const dropdown = byId('organizationKey');
+  dropdown.innerHTML = '';
+  if (organizations.length === 0) {
+    // do something to indicate that there are no organizations
```

Comment:
😅 
In this case, one possible thing would be to fall back to a text input. Might be considered "nice to have" since the organization can be set manually in the `settings.json` after creation.

Reply:
I agree, that would be great. I created a followup [hardening ticket](https://sonarsource.atlassian.net/browse/SLVSCODE-785) for this scenario, as it can be tackled independently and is not a blocker for now.